### ISSUE TITLE
Last build is missing websocket module

### ISSUE THREAD

**Issue by etienne-napoleone** [tomochain/tomomaster#133]
When running the docker image from last build, we receive an error during build:
```
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    | Error: Cannot find module 'websocket'
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Function.Module._resolveFilename (module.js:547:15)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Function.Module._load (module.js:474:25)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Module.require (module.js:596:17)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at require (internal/module.js:11:18)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Object.<anonymous> (/app/node_modules/web3-providers-ws/src/index.js:38:10)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Module._compile (module.js:652:30)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Object.Module._extensions..js (module.js:663:10)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Module.load (module.js:565:32)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at tryModuleLoad (module.js:505:12)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Function.Module._load (module.js:497:3)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Module.require (module.js:596:17)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at require (internal/module.js:11:18)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Object.<anonymous> (/app/node_modules/web3-core-requestmanager/src/index.js:53:24)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Module._compile (module.js:652:30)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Object.Module._extensions..js (module.js:663:10)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    |     at Module.load (module.js:565:32)
devnet_tomomaster.1.hiuibvpcdmd3@devnet.tomochain.com    | [nodemon] app crashed - waiting for file changes before starting...

```

**Comment by thanhson1085** [comment]
Yep, will check it

**Comment by thanhson1085** [comment]
It should be fixed by #134

**Comment by etienne-napoleone** [comment]
It is fixed, thanks
